chain/network/src/types.rs

```diff
@@ -53,6 +53,7 @@ pub struct KnownProducer {
 
 /// Ban reason.
 #[derive(borsh::BorshSerialize, borsh::BorshDeserialize, Debug, Clone, PartialEq, Eq, Copy)]
+#[borsh(use_discriminant = false)]
```

Comment:
I am glad we took another mile to fix that hidden footgun of "silently" changing the default from `false` to `true`.

@mina86 Thanks for calling that out!